.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,38 @@
+name: CI Workflow
+
+on:
+  push:
+    branches: [ '**' ]
+  workflow_dispatch:
+
+concurrency: ci-${{ github.ref }}
+
+jobs:
+  buildAndTest:
+    name: Build, Test, and Package
+    runs-on: ubuntu-latest
```

Comment:
@David-N-Perkins: other than the dependency vulnerabiliuty scan, is there any other operation we are missing from this `ci-workflow`?

Reply:
The vulnerability scan would go in different workflows. Sonar Cloud analysis could go in this workflow, but I put it in a different one for hri-mgmt-api.